ncm-spma/pom.xml

```diff
@@ -218,7 +247,45 @@
                 </sources>
                 <directoryIncluded>false</directoryIncluded>
               </mapping>
+              <mapping>
+                <directory>/usr/sbin</directory>
+                <filemode>755</filemode>
+                <username>root</username>
+                <groupname>root</groupname>
+                <directoryIncluded>false</directoryIncluded>
+                <sources>
+                  <source>
+                    <location>${project.build.directory}/sbin</location>
+                    <includes>
+                      <include>spma</include>
+                    </includes>
+                  </source>
+                </sources>
+                <configuration>false</configuration>
+                <documentation>false</documentation>
+              </mapping>
             </mappings>
+            <requires>
```

Comment:
current master has an improved way of organising the dependencies (since https://github.com/quattor/configuration-modules-core/pull/768). please rebase on master (maybe remove th ecurrent last commit), and move any extra depencies defined here a fine lines upwards where they are also defined